Dockerfile

```diff
@@ -1,15 +1,18 @@
+ARG tomcat_version=9.0.29
 
 FROM alpine
```

Comment:
any objections of using https://hub.docker.com/_/tomcat/ ?

Reply:
We choose build our image in that way because of the weight.
Our custom tomcat image is over 150 Mb, Original tomcat from DockerHub - over 500 Mb.

Reply:
BTW, we also have a dockerfile based on tomcat from DockerHub